packages/core/content-type-builder/admin/src/components/AttributeOptions/CustomFieldsList/index.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useCustomFields } from '@strapi/helper-plugin';
+import { Box } from '@strapi/design-system/Box';
+import { Grid, GridItem } from '@strapi/design-system/Grid';
+import { KeyboardNavigable } from '@strapi/design-system/KeyboardNavigable';
+import { Stack } from '@strapi/design-system/Stack';
+import { Link } from '@strapi/design-system/Link';
+import { useIntl } from 'react-intl';
+import EmptyAttributes from '../EmptyAttributes';
+import CustomFieldOption from '../CustomFieldOption';
+import getPadding from '../utils/getPadding';
+import { getTrad } from '../../../utils';
+
+const CustomFieldsList = () => {
+  const { formatMessage } = useIntl();
+  const customFields = useCustomFields();
+  const registeredCustomFields = Object.entries(customFields.getAll());
+
+  if (!registeredCustomFields.length) return <EmptyAttributes />;
```

Comment:
Can you avoid inline returns? There used to be an eslint rule about this in the admin...

Reply:
https://github.com/strapi/strapi/pull/14465/commits/323cf3bf24a56784c7ec3b91d15249c9c1dc00c1